lib/view_component/system_test_helpers.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module ViewComponent
+  module SystemTestHelpers
+    include TestHelpers
+    include Capybara::DSL
+
+    def page
```

Comment:
What do we need this for? I could see it conflicting with consumers' tests.

Reply:
It's been a while, but I recalled this was suggested earlier so that `assert` or any other capybara commands operate on `page`. Its a bit of syntactic sugar :) so you don't have to keep specifying `page` and can do something like:
```
assert find("[data-hidden-field]", visible: false) 
```
Without having to specify the `page`

Reply:
Interesting. I guess what I'm wondering now that I re-read this is whether we need a `capybara` dependency at all. I think we should try to avoid one. I think we can just have this feature provide a temporary URL to visit in whatever system test stack folks are using.

Reply:
Hey @joelhawksley; ideally, we want to avoid the dependency on general support. However, I feel like capybara is pretty standard, and I'am wondering if it makes sense to release with capybara support and then introduce wider support?

Reply:
@edwinthinks I think we included capybara at one point but that some folks had issues with it. I think that's why we have this section in the docs: https://viewcomponent.org/guide/testing.html#without-capybara. I think I'd prefer to not included `capybara` in the package. We can have it as a dev dependency in our test suite, though 👍🏻

Reply:
@joelhawksley picking up on this. I agree on the assessment on not including capybara as a dependency. I'am working on making the dependency only on our testing and not on the actual gem itself.